mettle/configurablepubsub/configurablepubsub.go

```diff
@@ -0,0 +1,87 @@
+// configurablepubsub is a wrapper around gpcpubsub package that allows us to configure num handlers
+package configurablepubsub
+
+import (
+	"context"
+	"fmt"
+	"net/url"
+	"path"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"gocloud.dev/gcp"
+	"gocloud.dev/pubsub"
+	"gocloud.dev/pubsub/gcppubsub"
+)
+
+const Scheme = "configurable-gcppubsub"
```

Comment:
That's a very long name for a scheme 😅

Reply:
haha I shorterned it to tm-gcppubsub. I wouldn't be against changing the package name to this also but the name is pretty descriptive.